profile/README.md

```diff
@@ -0,0 +1,7 @@
+# Well, hello there 👋
+
+Welcome to our little corner of GitHub! We are the Norwegian Labour and Welfare Directorate, "Arbeids- og velferdsdirektoratet", a part of the organization commonly known as [NAV](https://nav.no).
+
+We build our software in the open. It was funded by the taxpayers, so it should be freely available. We also strongly believe that open source is a great vehicle for collaboration and that it enhances the quality of our products.
```

Comment:
It was funded by the taxpayers -> It is funded by the taxpayers

Småpirk men vi er vel fortsatt betalt av skattebetalere?

Reply:
> Småpirk men vi er vel fortsatt betalt av skattebetalere?

Ja, jeg håper ikke de har slutta med det 😀